subprojects/gradle/runner/shared-test/src/main/kotlin/com/avito/runner/test/mock/MockDevice.kt

```diff
@@ -103,11 +103,12 @@ open class MockDevice(
     }
 
     override fun log(message: String) {
-        logger.log(message)
+        logger.info(message)
     }
 
     override fun notifyError(message: String, error: Throwable?) {
-        logger.notify(message, error)
+        val error: Throwable = error ?: java.lang.RuntimeException(message)
```

Comment:
`notifyError` signature looks strange
I consider adding more logging levels. Because If must have an error if you want call `notifyError`
`log(message)`
`warn(message)`
`error(message, throwable: Throwable)`

Reply:
changed to `warn` to make it consistent with logger